servicetalk-http-api/src/main/java/io/servicetalk/http/api/DefaultBlockingStreamingHttpRequest.java

```diff
@@ -154,6 +154,18 @@ public BlockingStreamingHttpRequest setQueryParameters(String key, String... val
         return this;
     }
 
+    @Nullable
+    @Override
+    public String fragment() {
+        return original.fragment();
+    }
+
+    @Override
+    public HttpRequestMetaData fragment(@Nullable String fragment) {
```

Comment:
But "setter" we need to override in all following classes because we have to change the return type:
1. HttpRequest
2. StreamingHttpRequest
3. BlockingStreamingHttpRequest
4. DefaultHttpRequestMetaData
5. DefaultHttpRequest
6. DefaultStreamingHttpRequest
7. DefaultBlockingStreamingHttpRequest